x-pack/plugins/serverless_search/public/application/components/overview_panels/select_client.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { PLUGIN_ID } from '../../../../common';
+
+import { useKibanaServices } from '../../hooks/use_kibana';
+import { LanguageDefinition } from '../languages/types';
+import { OverviewPanel } from './overview_panel';
+import { docLinks } from '../../../../common/doc_links';
+interface SelectClientProps {
+  setSelectedLanguage: (language: LanguageDefinition) => void;
+  languages: LanguageDefinition[];
+  selectedLanguage: LanguageDefinition;
+}
+export const SelectClientPanel: React.FC<SelectClientProps> = ({
+  setSelectedLanguage,
+  languages,
+  selectedLanguage,
+}) => {
+  const { http } = useKibanaServices();
+  const { euiTheme } = useEuiTheme();
+
+  const panelItems = languages.map((language) => (
+    <EuiPanel
+      hasBorder
+      borderRadius="m"
+      onClick={() => setSelectedLanguage(language)}
+      css={
+        selectedLanguage === language
+          ? css`
+              border: 1px solid ${euiTheme.colors.primary};
+            `
+          : css`
+              border: 1px solid ${euiTheme.colors.lightShade};
+            `
+      }
+      color={selectedLanguage === language ? 'primary' : 'plain'}
+    >
+      <EuiFlexGroup direction="column" justifyContent="center">
+        <EuiFlexItem>
+          <EuiImage
+            alt=""
+            src={http.basePath.prepend(`/plugins/${PLUGIN_ID}/assets/${language.iconType}`)}
+            height="32px"
+            width="32px"
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiText textAlign="center" color={selectedLanguage === language ? 'default' : 'subdued'}>
+            <h5>{language.name}</h5>
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiPanel>
```

Comment:
is there a reason we can't use [`EuiCard`](https://eui.elastic.co/#/display/card)? is an `EuiPanel` keyboard accessible?

Reply:
Each of these image is this different size, especially the curl.svg.  Adding in Card made the Curl image box look different from other boxes.  EuiPanel is the closest one that aligned with the design, which aligned each of these boxes similar( with little difference). 

I tested manually keyboard accessibility for EuiPanel. It works, but is there a better way to identify the keyboard accessibility for eui components?

Reply:
okay. and no that's how i test for keyboard accessibility too.

Reply:
With `EuiCard`:

<img width="1728" alt="Screenshot 2023-05-18 at 11 52 56 AM" src="https://github.com/elastic/kibana/assets/55930906/693f0b44-f082-4037-adee-d3a71d94bbe4">

Reply:
you can pass a react element as the image prop. does that fix the image rendering issues?

Reply:
Going in with Panel. Card doesn't allow to change the vertical gap between image and title. Thank you @sloanelybutsurely  for detailed review 🙌